build-farm/platform-specific-configurations/aix.sh

```diff
@@ -55,7 +55,7 @@ if [ "$JAVA_FEATURE_VERSION" -gt 11 ]; then
         # for the fallback mechanism, as downloading of the GA binary might
         # fail.
         set +e
-        wget -q -O - "${apiURL}" | tar xpzf - --strip-components=1 -C "$bootDir"
```

Comment:
Are you sure about removing `-` from the `wget` command? From the man page:

```
-O file
       --output-document=file
           The documents will not be written to the appropriate files, but all will be concatenated together and written to file.  If - is used as file, documents will be printed to standard output,
           disabling link conversion.  (Use ./- to print to a file literally named -.)
```

Reply:
Oh right, I thought it was a typo like the tar command, didn't realise it was doing something I'll add it back